api/src/services/pd_services.js

```diff
@@ -0,0 +1,26 @@
+let repo = require("../core/pd_repo");
```

Comment:
use import in all the class instead of require

Reply:
done